shared/reducers/fs.js

```diff
@@ -38,6 +39,15 @@ export default function(state: Types.State = initialState, action: FsGen.Actions
           // placeholder), which then gets updated when we hear back from RPC.
           return original
         }
+
+        toRemove = toRemove.concat(
+          original.children
+            .filter(child => !item.children.includes(child))
+            .toArray()
+            .map(name => Types.pathConcat(path, name))
+        )
+        console.log(`Removing entries in state.fs.pathItems: ${JSON.stringify(toRemove)}`)
```

Comment:
🔥

Reply:
Does it mean this is on fire or Song your pants will be on fire if you do this? I’m guessing later. But how do I log this? Just the array itself or construct an object that contains the array?